cypress/integration/Signup.spec.js

```diff
@@ -0,0 +1,53 @@
+/// <reference types="cypress" />
+
+import * as signUp from '../page-objects/Signup';
+
+const { internet } = require('faker');
+
+describe('Signup tests', () => {
+  beforeEach(() => {
+    cy.visit(Cypress.env('HOME_URL'));
+    signUp.signUpPage();
+  });
+
+  it('Should sign up succesfully', () => {
+    const id = Math.floor(Math.random() * 10000);
+    const username = `username${id}`;
+    signUp.addUsername(username); // internet.userName genera un usuario con fomato xxx.yyy y conduit no acepta ese formato
```

Comment:
all the code is in English, comments should also be in english. 
Put comments in the previos line, so lines are not so long. Linter should have a config to limit line lengths.